test/acceptance/tests/basic/basic_test.go

```diff
@@ -401,50 +386,59 @@ func TestValidation_EnvoyPublicListenerPort(t *testing.T) {
 	}
 }
 
-func TestValidation_ChecksVariable(t *testing.T) {
+func TestValidation_EnvoyReadinessPort(t *testing.T) {
 	t.Parallel()
 
 	cases := map[string]struct {
-		checksFile string
-		error      bool
+		port  int
+		error string
 	}{
-		"no-checks": {
-			checksFile: "test-no-checks.json",
+		"allowed-port": {
+			port: 23000,
 		},
-		"valid-checks": {
-			checksFile: "test-valid-checks.json",
+		"too-high-port": {
+			port:  65536,
+			error: "The envoy_readiness_port must be greater than 0 and less than or equal to 65535.",
 		},
-		"invalid-checks": {
-			checksFile: "test-invalid-checks.json",
-			error:      true,
+		"disallowed-port": {
+			port:  19000,
+			error: "The envoy_readiness_port must not conflict with the following ports that are reserved for Consul and Envoy",
+		},
+		"conflicts-with-listener-port": {
+			port:  20000,
+			error: "envoy_public_listener_port should not conflict with envoy_readiness_port",
 		},
 	}
 
 	terraformOptions := &terraform.Options{
-		TerraformDir: "./terraform/checks-validate",
+		TerraformDir: "./terraform/envoy-readiness-port-validate",
 		NoColor:      true,
 	}
 	terraform.Init(t, terraformOptions)
 
 	for name, c := range cases {
+		c := c
```

Comment:
(One day soon, we won't need this loop capture hack anymore: https://github.com/golang/go/wiki/LoopvarExperiment)